### ISSUE TITLE
Login Failed using Iphone XR IOS 16.2

### ISSUE THREAD

**Issue by SwayerPT** [hdnpt/geartrack-website#73]
Hello, a few days ago i have added a topic about this, and u said that u have fixed. But... is not fixed.
I will add a video and u cna watch and see by your eyes what really is happening when u try to Login via GMAIL.

basicly the session is failed and goes back to the login again. Could be a simple error or else.
On PC works perfectly. But i use Iphone so... i like to be sync.

https://user-images.githubusercontent.com/31244474/198575671-14b2eb07-8dd0-4562-b0f6-23f18b4ea891.mp4

**Comment by carlosflorencio** [comment]
This seems the behaviour of firebase being blocked due to restricted networks/proxys that buffer responses (corporate networks), does it work in mobile/home networks?

Can you send a message via the Crisp chat (should have been in the bottom right of that page 🤔 ) so that we can do a debug session together? (Thanks for all the feedback lately)

**Comment by SwayerPT** [comment]
No, even trying using Mobile Data or Home network with/without Another-Proxy. (8.8.8.8, 8.8.4.4).

I have a sugestion for you. 
Do u know STEAM APP right? They use the Website as APP template. You should do the same. You save time. 90% os the problems will be solved on website and you will have an APP xD.

**Comment by carlosflorencio** [comment]
Yeah using a hybrid technology (web in native) should be our choice when launching a mobile app. 

But that wouldn't solve this specific issue with the login 🤔,  can you check the developer console on the mobile to check for any logs/failed requests? For safari ios, you need to connect the iPhone via cable to the mac and use the Safari Technology Preview browser to inspect, I'd do this myself but I can't seem to replicate the issue.

**Comment by carlosflorencio** [comment]
Also, are you using an adblocker (via extension or dns like pihole/nextdns) ? If so, they might be blocking the crisp chat and some google domains (required for firebase).

**Comment by SwayerPT** [comment]
Wow... that could be the problem.. i am using one blocker... hummmm I will test soon.

**Comment by SwayerPT** [comment]
Nop, tested and AdBlock W/without is doing the same error. Probably sessionStorage error...
If you have an iphone, test it using ios16.2 Beta.

**Comment by carlosflorencio** [comment]
I don't participate on the iOS betas, with the adblock disabled, did you have the crisp chat button back?

**Comment by SwayerPT** [comment]
On IOS Safari does not show the Button. Only on PC Version

**Comment by carlosflorencio** [comment]
It should appear, your adblock should be blocking it.

**Comment by costa-alex** [comment]
I'm having the same issue since this morning on my iPhone (iOS 16.2).

On PC/Windows it is working fine.

**Comment by carlosflorencio** [comment]
Hey @costa-alex , thanks for reporting it.

I can't replicate atm (I'm updating my phone to 16.1.2), can you provide your full user-agent?

I'm using this regex `/Version\/16.\d+ Mobile.*Safari/i` to match the Safari versions where we need to use a different login flow (workaround for this [issue in firebase](https://github.com/firebase/firebase-js-sdk/issues/6716)), maybe it's missing something.

**Comment by costa-alex** [comment]
My bad, I’m using 16.1.2 not 16.2.

User agent: 
`Mozilla/5.0 (iPhone; CPU iPhone OS 16_1 like Mac OS X) AppleWebKit/605.1.15 (KHTML, like Gecko) CriOS/108.0.5359.52 Mobile/15E148 Safari/604.1`

**Comment by carlosflorencio** [comment]
Should be working now, can you test @costa-alex ?

**Comment by costa-alex** [comment]
@carlosflorencio yes, it’s working. Thanks
